src/containers/profile/timeline.svelte

```diff
@@ -1,49 +1,96 @@
 <script>
   import { createEventDispatcher } from 'svelte';
-  import Button from 'ui/button.svelte';
+  import { Button, Loading } from 'attractions';
+  import EmptyState from '../../components/common/empty-state.svelte';
```

Comment:
At least be consistent until the alias is replaced with something else so that it will be easier to use _find and replace all_
```suggestion
  import EmptyState from '@/components/common/empty-state.svelte';
```